runtime/parachains/src/configuration/migration.rs

```diff
@@ -17,23 +17,20 @@
 //! A module that is responsible for migration of storage.
 
 use crate::configuration::{self, Config, Pallet, Store, MAX_POV_SIZE};
-use frame_support::{
-	pallet_prelude::*,
-	traits::StorageVersion,
-	weights::{OldWeight, Weight},
-};
+use frame_support::{pallet_prelude::*, traits::StorageVersion, weights::Weight};
 use frame_system::pallet_prelude::BlockNumberFor;
 
 /// The current storage version.
 ///
 /// v0-v1: <https://github.com/paritytech/polkadot/pull/3575>
 /// v1-v2: <https://github.com/paritytech/polkadot/pull/4420>
 /// v2-v3: <https://github.com/paritytech/polkadot/pull/6091>
-pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(3);
+/// v3-v4: <https://github.com/paritytech/polkadot/pull/6345>
+pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(4);
 
-pub mod v3 {
+pub mod v4 {
```

Comment:
I'm not sure if I have to add this migration to the runtime or it should be done just before the runtime release?

At the moment it's not added to any of the runtimes.

Reply:
@eskimor from the list above: the test is done, #6271 seems to be on hold for now.
As discussed with @ordian - the migrations for disputes and HostConfig are added in the runtimes for all networks - polkadot, kusama, westend, rococo (which is also versi).

Anything else we need to do before merging?